### PR TITLE
Fix/bert example bug

### PR DESCRIPTION
**Pull request by pranavpsv**
Fixed bert_for_sequence_classification.py bug to return logits properly using Bert's pooled output.
Used huggingface/transformers tokenizer's encode_plus function to encode text (which provides attention_mask as well).
Tested to ensure it was working using thufeifeibear/turbo_transformers_cpu:latest image.

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/TurboTransformers?pullRequest=155) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/TurboTransformers?pullRequest=155) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/TurboTransformers?pullRequest=155) it.</sub>
